docs/product/components/expandable.html

```diff
@@ -31,7 +31,7 @@
                 <button class="s-btn s-btn__filled mb8"
                     data-controller="s-expandable-control"
                     data-s-expandable-control-toggle-class="is-selected"
-                    aria-expanded="true"
+                    aria-expanded="false"
```

Comment:
I think this is the right idea in the wrong place.  `aria-expanded="true"` should always be paired with `s-expandable is-expanded` (and in this case `s-btn is-selected`).

Right now this code is behaving as expected, but the code above between `{%highlight html %}` and `{% endhighlight %}` is wrong for the reasons I described above.  They would be fixed either by setting `aria-expanded="false"` there, or applying the two additional classes to the appropriate elements.

Reply:
Oh carp, my bad.  This is correct, but we should update the code snippet as well.